src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentsLoadAction.java

```diff
@@ -29,6 +29,7 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         String courseId = getRequestParamValue(Const.ParamsNames.COURSE_ID);
         Assumption.assertNotNull(courseId);
         String fsname = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
+        int fsindex = Integer.parseInt(getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_INDEX));
```

Comment:
Need to verify the param is not null;
then parse it with try-catch; if fail to parse, fast-fail it.